docs/user-guide/recipes.ipynb

```diff
@@ -0,0 +1,139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d7608bb4-d7ce-4a36-8021-f1a52ad27ec5",
+   "metadata": {},
+   "source": [
+    "# Recipes"
```

Comment:
Should this be a sibling of `User Guide`, so it can be reached more easily in fewer clicks?

Reply:
Can be. We have limited real estate in the top bar. Do you think this is important / commonly used enough?

Reply:
I thought so, yes?

Reply:
Done